From a55f5485c65ffe1c7812b3722eaa36c15bc63385 Mon Sep 17 00:00:00 2001
From: Eugene Syromyatnikov <evgsyr@gmail.com>
Date: Mon, 24 Sep 2018 00:43:52 +0200
Subject: [PATCH 182/293] xlat: update kvm_cap

* xlat/kvm_cap.in (KVM_CAP_NESTED_STATE): New constant, introduced
by Linux commit v4.19-rc1~87^2~62.
(KVM_CAP_ARM_INJECT_SERROR_ESR): New constant, introduced by Linux
commit v4.19-rc1~55^2~12^2~14, merged with its final value
in v4.19-rc1~55^2~12.
(KVM_CAP_MSR_PLATFORM_INFO): New constant, introduced by Linux commit
v4.19-rc5~8^2~2.
(KVM_CAP_PPC_NESTED_HV): New constant, introduced by Linux commit
v4.20-rc1~113^2~107^2~1.
(KVM_CAP_HYPERV_SEND_IPI): New constant, introduced by Linux commit
v4.20-rc1~113^2~72.
KVM_CAP_COALESCED_PIO): New constant, introduced by Linux commit
v4.20-rc1~113^2~31.
(KVM_CAP_HYPERV_ENLIGHTENED_VMCS): New constant, introduced by Linux
commit v4.20-rc1~113^2~25.
(KVM_CAP_EXCEPTION_PAYLOAD): New constant, introduced by Linux commit
v4.20-rc1~113^2~8.
(KVM_CAP_ARM_VM_IPA_SIZE): New constant, introduced by Linux commit
v4.20-rc1~113^2~5^2~8 and updated by Linux commit
v4.20-rc1~113^2~5.
* NEWS: Mention this.

Co-Authored-by: Dmitry V. Levin <ldv@altlinux.org>
---
 xlat/kvm_cap.in | 9 +++++++++
 2 files changed, 11 insertions(+), 2 deletions(-)

 * Bug fixes
diff --git a/xlat/kvm_cap.in b/xlat/kvm_cap.in
index a3819e7..6d229e0 100644
--- a/xlat/kvm_cap.in
+++ b/xlat/kvm_cap.in
@@ -150,3 +150,12 @@ KVM_CAP_GET_MSR_FEATURES		153
 KVM_CAP_HYPERV_EVENTFD			154
 KVM_CAP_HYPERV_TLBFLUSH			155
 KVM_CAP_S390_HPAGE_1M			156
+KVM_CAP_NESTED_STATE			157
+KVM_CAP_ARM_INJECT_SERROR_ESR		158
+KVM_CAP_MSR_PLATFORM_INFO		159
+KVM_CAP_PPC_NESTED_HV			160
+KVM_CAP_HYPERV_SEND_IPI			161
+KVM_CAP_COALESCED_PIO			162
+KVM_CAP_HYPERV_ENLIGHTENED_VMCS		163
+KVM_CAP_EXCEPTION_PAYLOAD		164
+KVM_CAP_ARM_VM_IPA_SIZE			165
-- 
1.7.12.4

